app/styles/project/publicatie.scss

```diff
@@ -0,0 +1,283 @@
+/* ==========================================================================
+  Publicatie
+  Style what publicaties looks like here.
+ ========================================================================== */
+
+/* Style document publish for besluitenlijst, remove when fixed bug
+ ========================================================================== */
+
+.au-c-document-publish {
+  [typeof="besluit:BehandelingVanAgendapunt"],
+  [property="besluit:behandelt"] {
+    margin-bottom: 2rem;
+    display: block;
+  }
+
+  [typeof="besluit:BehandelingVanAgendapunt"]:not([property="ext:behandelt"]) {
+    [property="besluit:openbaar"][content="true"] {
+      &:before {
+        content: "Agendapunt - "
```

Comment:
There is probably a reason for this, but I think it's weird to add real content this way.

Reply:
Yes, not a fan of it either but not in scope to fix this now. The publication templates need to be reviewed to find a better solution.